source/docs/software/commandbased/commands-v3/index.rst

```diff
@@ -0,0 +1,110 @@
+# Commands v3 Programming
+
+.. note:: Commands v3 is currently in development for the 2027 season. This documentation is a work in progress.
+
+This section serves as an introduction to and reference for the WPILib Commands v3 framework.
+
+Commands v3 is the next-generation command-based framework for Java that introduces imperative-style command writing using coroutines. Instead of building complex chains of lambda expressions and decorators, you can write command logic as straightforward sequential code with loops, conditionals, and explicit control flow.
+
+**WPILib recommends Commands v3 for Java teams starting in 2027.** Commands v3 is actively being developed with new features including improved telemetry, enhanced trigger functionality, and priority-based interruption.
```

Comment:
I recommend deleting this paragraph. It's essentially duplicative of the should you use Commands v3. And it's way too early to actively recommend it before any third party has even tried it. Alternately, it could be reworded as a request for java teams to alpha test it.